tests/testdata/generate.rs

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env -S cargo -Z script
+---cargo
+[package]
+edition = "2021"
+[dependencies]
+rcgen = { version = "0.13", features = ["pem"] }
+---
+
+use std::fs::OpenOptions;
+use std::io::Write;
+use std::env;
+
+use rcgen::{DistinguishedName, KeyPair, DnType, IsCa, BasicConstraints};
+
+fn main() {
+    let ca_key = KeyPair::generate().unwrap();
+    let mut ca_params = rcgen::CertificateParams::default();
+    let mut distinguished_name = DistinguishedName::new();
+    distinguished_name.push(
+        DnType::CommonName,
+        "Pebble CA".to_string(),
+    );
+    ca_params.distinguished_name = distinguished_name;
```

Comment:
Nit: initialize `ca_params` after setting up `distinguished_name`, instead of the interspersing that happens here?

Reply:
Done.

Reply:
Hmm, maybe I got confused. Looks okay to me!